src/themes/default/components/core/blocks/Compare/ProductAttribute.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <span>{{value|htmlDecode}}</span>
+</template>
+
+<script>
+  import { coreComponent } from 'lib/themes'
+
+  export default {
+    data () {
+      return {
+        value: ''
+      }
+    },
+    beforeMount () {
+      let parsedValues = this.product[this.attribute.attribute_code]
+
+      if (!parsedValues) {
+        this.value = this.emptyPlaceholder
+      } else {
+        parsedValues = parsedValues.split(',')
+        let results = []
+        for (let parsedVal of parsedValues) {
+          if (this.attribute.options) {
+            let option = this.attribute.options.find(av => {
+              /* eslint eqeqeq: "off" */
+              return av.value == parsedVal
+            })
+            if (option) {
+              results.push(option.label)
+            }
+          } else {
+            results.push(parsedVal)
+          }
+        }
+        this.value = results.join(', ')
+      }
```

Comment:
This seems like a copy of the logic included in ProductAttribute component (sorry if I’ve given the wrong name of the component / I meant the attribute component used on product pages).

Maybe it’s good idea to: use mixin of the component mentioned above OR move the value selection logic to helper module and use it within both components? 

We should avoid DRY violations

Reply:
ProductAttribute component extended